MoreLinq/Batch.cs

```diff
@@ -50,7 +50,7 @@ static partial class MoreEnumerable
 
         public static IEnumerable<IEnumerable<TSource>> Batch<TSource>(this IEnumerable<TSource> source, int size)
         {
-            return Batch(source, size, x => x);
+            return Batch(source, size, Identity<IEnumerable<TSource>>);
```

Comment:
Type argument is redundant:

```suggestion
            return Batch(source, size, Identity);
```